instantly/parse.go

```diff
@@ -0,0 +1,59 @@
+package instantly
+
+import (
+	"strconv"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/jsonquery"
+	"github.com/amp-labs/connectors/common/urlbuilder"
+	"github.com/spyzhov/ajson"
+)
+
+func makeGetTotalSize(dataLocation string) common.ListSizeFunc {
```

Comment:
again, this should not be defined for every connector and same for all other functions in this file. Maybe except for the `makeNextRecordsURL` since url are specific to the  provider.

Reply:
I went further. You are right that there is repitition, in this case this method shouldn't even exist. Here is a PR: https://github.com/amp-labs/connectors/pull/991